application.py

```diff
@@ -117,28 +120,30 @@ class Meta: #all the variables you want to see
 movie_schema = MovieSchema()
 movies_schema = MovieSchema(many=True)
 
-# # User In Group Class
-# class UserGroup(db.Model):
-#     groupID = db.Column(db.Integer, primary_key=True)
-#     userID = db.Column(db.String(200), nullable=False)
-#     joined_at = db.Column(db.Integer)
-#     left_at = db.Column(db.String(200))
-
-#     def __init__(self,movieID,name,year,director):
-#         self.movieID = movieID
-#         self.name = name
-#         self.year = year
-#         self.director = director
-
-# # UserGroup Schema - for marshmallow
-# class UserGroupSchema(ma.Schema):
-#     class Meta: #all the variables you want to see
-#         strict = True
-#         fields = ('movieID','name','year','director')
-
-# # Initiate Movie Schema
-# usergroup_schema = UserGroupSchema()
-# usergroups_schema = UserGroupSchema(many=True)
+
+class Userratesmovie(db.Model): ##
+    movieID = db.Column (db.Integer, db.ForeignKey(Movie.movieID),primary_key=True)
+    userID = db.Column (db.Integer, db.ForeignKey(User.userID),primary_key=True)
+    isLiked = db.Column (db.Boolean, nullable = False)
+
+    #user = db.relationship(User, backref=db.backref("ratings", cascade="all, delete-orphan"))
```

Comment:
If this is no longer used, I think we should delete this to avoid any confusion. Thoughts?

Reply:
I'm okay with deleting it if that makes the code cleaner and clearer.  This piece of code was originally left in in case it would be useful when creating the rest of the relationship endpoints in Flask that will be made in Sprint 2.  However, as I reflect on this, I agree that it may be best to remove it since we will not have relationship that will use this different approach.

Reply:
Have deleted the line. Went through the code and found a few more similar pieces of redundant code.  They have also been removed.  Thanks for pointing this out